README.md

```diff
@@ -253,6 +253,15 @@ DotNet:
 instrumentation.opentelemetry.io/inject-dotnet: "true"
 ```
 
+Go:
+
+Go auto-instrumentation also honors an annotation that will be used to set the [OTEL_GO_AUTO_TARGET_EXE env var](https://github.com/open-telemetry/opentelemetry-go-instrumentation/blob/main/docs/how-it-works.md).
```

Comment:
is this env var mandatory? 

What happens if it is not set? Is there any default value? I think we should document this in the CRD.